CONTRIBUTORS

```diff
@@ -32,6 +32,7 @@ Shahoyan, Ruben
 Stockmanns, Tobias
 Ustyuzhanin, Andrey
 von Haller, Barthélémy
+Wang, Yanzhao
```

Comment:
BTW, if you have an ORCiD, append it like this
`Family, Give [https://orcid.org/0000-XXXX-XXXX-XXXX]`.

Reply:
Ok, thanks for the info.